kodilla-patterns2/src/main/java/com/kodilla/patterns2/adapter/bookclasifier/libraryb/Statistics.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.patterns2.adapter.bookclasifier.libraryb;
+
+import java.util.Arrays;
+import java.util.Map;
+
+public class Statistics implements BookStatistics {
+    @Override
+    public int averagePublicationYear(Map<BookSignature, Book> books) {
+        if (books.size() == 0) return 0;
+        int sum = 0;
+        for (Map.Entry<BookSignature, Book> entry : books.entrySet()) {
```

Comment:
a może by tak streama zasadzić?

Reply:
To akurat jest kod podany w instrukcji - stan zastany na którym miałem pracować. Ale zmieniłem na stream :)